src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md

```diff
@@ -6,15 +6,11 @@ toc_not_nested: true
 docs_area: deploy
 ---
 
-This page provides information about CockroachDB {{ site.data.products.dedicated }} clusters on Microsoft Azure, including frequently asked questions and limitations during [limited access](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/cockroachdb-feature-availability). To create a cluster, refer to [Create Your Cluster]({% link cockroachcloud/create-your-cluster.md %}).
-
-{{site.data.alerts.callout_info}}
-{% include feature-phases/azure-limited-access.md %}
-{{site.data.alerts.end}}
+This page provides information about CockroachDB {{ site.data.products.dedicated }} clusters on Microsoft Azure, including frequently asked questions and limitations. To create a cluster, refer to [Create Your Cluster]({% link cockroachcloud/create-your-cluster.md %}).
 
 ## Limitations
 
-During [limited access](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/cockroachdb-feature-availability), CockroachDB {{ site.data.products.dedicated }} clusters on Azure have the following temporary limitations. To express interest or request more information about a given limitation, contact your Cockroach Labs account team. For more details, refer to the [FAQs](#faqs).
+CockroachDB {{ site.data.products.dedicated }} clusters on Azure have the following temporary limitations. To express interest or request more information about a given limitation, contact your Cockroach Labs account team. For more details, refer to the [FAQs](#faqs).
 
 ### Regions
```

Comment:
This appears to be slightly different from the regions page documentation on azure. Suggest using an include or linking out

Reply:
Removed the specific list here and pointed to the main Regions topic.